### DISCUSSION TITLE
Add favorite button on presentation image

### DISCUSSION THREAD

**Discussion by eerison** [photoview/photoview#895] [Ideas]
Hello, I was wondering if it could be a good idea add option to favorite a image when I am visualising it on full screen.

sometimes I am sliding the photos and I would like to favorite it, without exit the slide.

maybe you could list here where the current favorite button is located and when in what file appear the image in fullscreen.

**Comment by kkovaletp** [comment]
I'd like to see there not only the Favorite button but also the Info button

**Comment by eerison** [comment]
It would be nice as well,  but maybe it could be 2 prs.
